packages/energy-api-influxdb/test/app.e2e-spec.ts

```diff
@@ -154,4 +154,28 @@ describe('ReadsController (e2e)', () => {
         expect(reads[0].value).to.equal((2500 - 1500) * 10 ** 3);
       });
   });
+
+  it('should return last meter read', async () => {
```

Comment:
This test case does not really show on how the /latest functionality works, I suggest adding multiple reads with different dates and then use the latest to retrieve the oldest (?) one.

Reply:
Yeah, good point.

Reply:
> This test case does not really show on how the /latest functionality works, I suggest adding multiple reads with different dates and then use the latest to retrieve the oldest (?) one.

OK. I see there are other reader added during `before(async () => {` step.